source/_integrations/zha.markdown

```diff
@@ -583,6 +583,14 @@ Since all Zigbee Coordinator radio adapters are very sensitive/susceptible to al
   - Place your Zigbee Coordinator away from any Wi-Fi access points and all other sources of WiFi.
   - Wi-Fi frequency ranges can overlap with Zigbee, see the section above on defining Zigbee channel use.
 
+### Problems upgrading Zigbee device firmware via OTA
+
+Be aware that OTAU (Over-The-Air Upgrade) of Zigbee devices typically takes around 10 minutes. While a device is updating a lot of traffic is generated on the network, therefore it is recommended to not execute multiple updates at the same time. Always check that battery-powered devices have at least 70% of battery-life left before starting a firmware upgrade (a general tip is therefore to simply put new fresh batteries in before upgrading any OTA firmware upgrade), OTA updating of firmware is very power-consuming and while some devices do check for a minimum battery-level before starting the upgrade and might refuse to update unless have enough battery-life left, not all do so.
```

Comment:
> therefore it is recommended to not execute multiple updates at the same time.

This is not something we should burden our users with. If this is an issue it should be fixed.

Reply:
> Always check that battery-powered devices have at least 70% of battery-life left before starting a firmware upgrade

Same.